Calibration/PPSAlCaRecoProducer/test/hlt_out_singleRAW.py

```diff
@@ -0,0 +1,293 @@
+# hltGetConfiguration /users/jjhollar/ImportAlcaPathsTry2/V4 --globaltag auto:run3_hlt --setup /dev/CMSSW_12_1_0/GRun/V14 --data --unprescale --customise HLTrigger/Configuration/customizeHLTforCMSSW.customiseFor2018Input
+
+# /users/jjhollar/ImportAlcaPathsTry2/V4 (CMSSW_12_1_0)
+
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process( "HLTX" )
+process.load("setup_dev_CMSSW_12_1_0_GRun_V14_cff")
```

Comment:
I wonder: do you really need to dump in the release this whole (old) version of the GRun menu? It will become obsolete rather soon, and probably not even able to be run in the release: by the way, does this test run in the master?
For a test, shouldn't you preferably only consider your Alca paths, disregarding the other ones?

Reply:
@grzanka what's your take of this comment?

Reply:
@perrotta I can modify the `hlt_out_singleRAW.py` file so that it could generate the GRun file on its own (if needed), like here:
```Python
import os
import subprocess

if 'setup_dev_CMSSW_12_1_0_GRun_V14_cff.py' not in os.listdir():
    config_command = 'hltGetConfiguration /users/jjhollar/ImportAlcaPathsTry2/V4 --globaltag auto:run3_hlt --setup /dev/CMSSW_12_1_0/GRun/V14 --data --unprescale --customise HLTrigger/Configuration/customizeHLTforCMSSW.customiseFor2018Input'
    subprocess.run(config_command, shell=True, check=True)
```
Can you think of any better way to do this?

For the second part, do you mean the `/eos/user/m/maaraujo/HLT_work/HLT_rem_322022/Skim_Run322022_LS1025to1344_1.root` file? I can place it in a directory similar to what @mmusich suggested here https://github.com/cms-sw/cmssw/pull/36273#discussion_r804922738 - e.g. `/eos/cms/store/group/alca_global/pps_alcareco_producer_tests` (just something else instead of "unit-tests")

Reply:
I guess I'll go with this solution. Another way would be to put `setup_dev_CMSSW_12_1_0_GRun_V14_cff.py` e.g. in `/eos/cms/store/group/alca_global/pps_alcareco_producer_tests`, but, since @perrotta said it might quickly become obsolete (or already is), I think generating it when needed is a better idea.

Anyway, there is also the `Skim_Run322022_LS1025to1344_1.root` file which is supposed to be placed in some AlCa path (if I correctly understood what @perrotta wrote). As I don't have the permissions, @tvami could you copy `/eos/user/k/kocotm/Shared/alcarecoproducer/Skim_Run322022_LS1025to1344_1.root` to `/eos/cms/store/group/alca_global/pps_alcareco_producer_tests`? It might take some time - the file is almost 5 GB!

Reply:
As I said, it is an overkill to dump a whole GRun menu only to access one (or a few) AlCa paths that you need for your test: please move to a lighter menu, and possibly a less obsolete version of it